.github/workflows/tests.yml

```diff
@@ -236,7 +235,6 @@ jobs:
           - sytest-tag: buster
             postgres: postgres
             workers: workers
-            redis: redis
```

Comment:
Noting that this now is only different from the previous run in the list by using `postgres` instead of `multi-postgres`. Happy to accept that this may still be worth testing though.

Reply:
Correct! Maybe that is not worth testing though, would be worth hearing other's thoughts.

Reply:
According to @erikjohnston:

> Ah, OK. I think that makes `(buster, postgres, workers, redis)` pretty redundant, compared with `(buster, multi-postgres, workers)`